e2e/testcases/correction-birth/correct-birth-record-3.spec.ts

```diff
@@ -0,0 +1,1200 @@
+         * Expected result: should
+         * - redirect to mother's details page
+         * - focus on mother's id
+         */
+
+        expect(page.url().includes('correction')).toBeTruthy()
+        expect(page.url().includes('mother-view-group')).toBeTruthy()
+        expect(page.url().includes('#motherPassport')).toBeTruthy()
+
+        await page.locator('#motherPassport').fill(updatedMotherDetails.id)
+
+        await page.waitForTimeout(500)
+
+        await page.getByRole('button', { name: 'Back to review' }).click()
+
+        /*
+         * Expected result: should
+         * - redirect to review page
+         * - show previous id with strikethrough
+         * - show updated id
+         */
+
+        expect(page.url().includes('correction')).toBeTruthy()
+        expect(page.url().includes('review')).toBeTruthy()
+
+        await expect(
+          page.locator('#mother-content #ID').getByText(updatedMotherDetails.id)
+        ).toBeVisible()
+      })
+
+      test('3.4.6 Change usual place of residence', async () => {
+        await page
+          .locator('#mother-content #Usual')
+          .getByRole('button', { name: 'Change', exact: true })
+          .click()
+
+        /*
+         * Expected result: should
+         * - redirect to mother's details page
+         * - focus on mother's Usual place of resiedence
+         */
+
+        expect(page.url().includes('correction')).toBeTruthy()
+        expect(page.url().includes('mother-view-group')).toBeTruthy()
+        expect(page.url().includes('#countryPrimary')).toBeTruthy()
+
+        await page.locator('#statePrimaryMother').click()
+        await page.getByText(updatedMotherDetails.address.province).click()
+
+        await page.locator('#districtPrimaryMother').click()
+        await page.getByText(updatedMotherDetails.address.district).click()
+
+        await page
+          .locator('#cityPrimaryMother')
+          .fill(updatedMotherDetails.address.town)
+
+        await page
+          .locator('#addressLine1UrbanOptionPrimaryMother')
+          .fill(updatedMotherDetails.address.residentialArea)
+
+        await page
+          .locator('#addressLine2UrbanOptionPrimaryMother')
+          .fill(updatedMotherDetails.address.street)
+
+        await page
+          .locator('#addressLine3UrbanOptionPrimaryMother')
+          .fill(updatedMotherDetails.address.number)
+
+        await page
+          .locator('#postalCodePrimaryMother')
+          .fill(updatedMotherDetails.address.zipCode)
+
+        await page.waitForTimeout(500)
+
+        await page.getByRole('button', { name: 'Back to review' }).click()
+
+        /*
+         * Expected result: should
+         * - redirect to review page
+         * - show previous Usual place of resiedence with strikethrough
+         * - show updated Usual place of resiedence
+         */
+
+        expect(page.url().includes('correction')).toBeTruthy()
+        expect(page.url().includes('review')).toBeTruthy()
```

Comment:
Do we need to ignore case? we could use the same pattern as in `e2e/testcases/death/declaration/death-declaration-11.spec.ts`